src/items/implementations.md

```diff
@@ -50,10 +50,44 @@ bracketed set of associable items.
 The nominal type is called the _implementing type_ and the associable items are
 the _associated items_ to the implementing type.
 
-Inherent implementations associate the contained items to the implementing type.
-The associated item has a path of a path to the implementing type followed by
-the associate item's path component. Inherent implementations cannot contain
-associated type aliases.
+Inherent implementations associate the contained items to the
+implementing type.  Inherent implementations can contain [associated
+functions] (including methods), [associated types], and [associated
+constants]. They cannot contain associated type aliases.
+
+[associated functions]: associated-items.html#associated-functions-and-methods
+[associated types]: associated-items.html#associated-types
+[associated constants]: associated-items.html#associated-constants
+
+The path to an associated item is: any path to the implementing type,
```

Comment:
It would be nice if the first "path" was a link to the chapter on paths. I wouldn't block the PR on it though.

Reply:
done I think.  Unless I linked to the wrong place.